### PR TITLE
[#111344356]Added trailing backslash to PSMoudlePath during msi installation.

### PR DESCRIPTION
**Pull request by haocs**
Added trailing backslash to PSMoudlePath during msi installation.

### CODE REVIEW

**Comment by markcowl** [comment]
msi run here:  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/596/

**Comment by haocs** [comment]
new msi job - http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/597/

File: src/Common/Commands.Common/ServiceClientTracingListener.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.WindowsAzure.Commands.Common
+{
+    class ServiceClientTracingListener : IServiceClientTracingInterceptor
```
Comment (markcowl) [inline src/Common/Commands.Common/ServiceClientTracingListener.cs]: Why *listener", why not make this class ServiceClientTracingInterceptor?

**Comment by markcowl** [comment]
@haocs If possible, we should add in amock test that verifies debug messages are written to the queue

**Pull request merged**